kubernetes/database.libsonnet

```diff
@@ -37,6 +37,8 @@ local k = import 'kubernetes/kube.libsonnet';
     local this = self,
     // bento is the bento which contains the database cluster and application
     bento:: error 'bento is required',
+    // database_name is the name of the application database to create within the cluster
+    database_name:: error 'database_name is required',
```

Comment:
is marking this required going to cause issues for folks?

Reply:
It should be fine. This is one the `PostgresqlClusterServiceAssignment`, which isn't used yet. You were probably thinking this was the `PostgresqlDatabaseCluster`.